proto/blob/rollkit/blob.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package rollkit.blob;
+
+option go_package = "github.com/rollkit/celestia-openrpc/proto/rollkit/blob";
```

Comment:
why do you need `rollkit` for the path? can this be made rollkit independent?

Reply:
It's not related to `rollkit`, it can be renamed 👍🏽

Reply:
renamed to `celestia.openrpc`